log/logs.log

```diff
@@ -0,0 +1,9 @@
+2023-04-09 18:20:44,179 INFO mate.academy.service.AuthenticationServiceImpl:13 Method login was called. Params: login = bob
+2023-04-09 18:20:44,184 INFO mate.academy.service.OrderServiceImpl:17 Method completeOrder was called. Param: userId = 2
```

Comment:
something wrong) gitignore does not work. Do you check files before committing? You must always do it.

![image](https://user-images.githubusercontent.com/20326858/230783589-2f4f45e8-6b7d-41d1-b554-d840ee08f6a2.png)